ingestion/src/metadata/profiler/orm/functions/table_metric_computer.py

```diff
@@ -309,29 +340,25 @@ def table_storage(self):
             where_clause,
         )
 
-        res = self.runner._session.execute(query).first()
+        res = self.runner._session.execute(query).mappings().all()
```

Comment:
BigQuery can have virtually infinite tables so there is an edge case where the process hosting the ingestion cannot contain this lise. Therefore this needs to be bound in some form. I can suggest some filtering strategies:

1. Grabbing a window "around" the specific table using "order by" and "between".
2. Using hash based filters.

In any case the cache should be by table id and not for the full `__TABLES__` object so that individual tables can be inserted and evicted based on usage.